hefquin-engine/src/main/java/se/liu/ida/hefquin/engine/queryplan/physical/impl/PhysicalOpFilter.java

```diff
@@ -41,4 +44,23 @@ public LogicalOpFilter getLogicalOperator() {
 		return lop;
 	}
 
+	public static class Factory implements PhysicalOpFactory
+	{
+		@Override
+		public boolean supports( final LogicalOperator lop, final ExpectedVariables inputVars ) {
+			if( lop instanceof LogicalOpFilter ){
+				return true;
+			}
+			return false;
```

Comment:
Maybe simplify as follows:
```suggestion
			return ( lop instanceof LogicalOpFilter );
```

Reply:
This one is applicable also in several other cases here.